src/components/WatchList/styleds.tsx

```diff
@@ -58,12 +58,12 @@ export const RowWrapper = styled(Box)<{ isSelected: boolean }>`
   border-bottom: 1px solid ${({ theme }) => theme.text9};
   cursor: pointer;
   border-radius: 4px;
-  background-color: ${({ theme, isSelected }) => (isSelected ? theme.color3 : theme.color2)};
+  background-color: ${({ theme, isSelected }) => (isSelected ? theme.bg6 : theme.color2)};
 
   height: 64px;
 
   &:hover {
-    background-color: ${({ theme }) => theme.color3};
+    background-color: ${({ theme }) => theme.bg6};
```

Comment:
I can change any `color*` to eerieBlack in dark mode and white in light mode?

Reply:
yes 

![Screenshot 2022-06-28 at 5 46 38 PM](https://user-images.githubusercontent.com/6604146/176176291-f44caf2a-89b2-44cb-93f9-c6ccb275a7c5.png)

based on above screenshot, we have
Eerie Black <-> Platinum
Eerie Black <-> White

now we have `color3` which is for `Eerie Black <-> Platinum`

you can add `color10` which will be for `Eerie Black <-> White`

as of now storybook works only on light mode so that color10 will be White

and you will also need to update interface to add `color10`

Reply:
Done, and added in interface https://github.com/pangolindex/interface/pull/549